src/locales/locale-en.json

```
@@ -514,5 +514,9 @@
     "HEADER_LINK": "foodsaving worldwide community",
     "LAST_UPDATED": "Last updated {relativeDate}"
   },
+  "INACTIVITY": {
+    "WHAT": "What does it mean to be inactive?",
+    "HELP": "Everyone who did not log into the group for 30 days or more counts as inactive. These users will not receive notification emails anymore."
```

```suggestion
    "HELP": "Everyone who did not visit the group for {dayCount} days or more counts as inactive. These users will not receive notification emails anymore."
```

- a group visit counts as activity
- make it parameterized so we can change the setting without invalidating the translation message

---

Yes, right, makes sense.